src/Models/QuoteItemOption.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Rapidez\Core\Models;
+
+use Illuminate\Database\Eloquent\Casts\Attribute;
+use Illuminate\Support\Facades\DB;
+
+class QuoteItemOption extends Model
+{
+    protected $table = 'quote_item_option';
+
+    protected $primaryKey = 'option_id';
+
+    public $timestamps = false;
+
+    protected $appends = ['label'];
+
+    public function quote_item()
+    {
+        return $this->belongsTo(config('rapidez.models.quote_item'), 'item_id');
+    }
+
+    protected function value(): Attribute
+    {
+        return Attribute::make(
+            get: fn (string $value) => match ($this->code) {
+                'info_buyRequest' => json_decode($value),
+                'option_ids'      => explode(',', $value),
+                default           => (function () use ($value) {
+                    if (in_array($this->option->type, ['drop_down'])) {
+                        return config('rapidez.models.product_option_type_value')::find($value)
+                            ->titles()
+                            ->whereIn('store_id', [0, config('rapidez.store')])
+                            ->orderByDesc('store_id')
+                            ->first()->title;
+                    }
+
+                    if ($this->option->type == 'file') {
+                        return json_decode($value);
+                    }
+
+                    return $value;
+                })()
+            })->shouldCache();
+    }
+
+    protected function label(): Attribute
+    {
+        return Attribute::make(
+            get: fn () => $this
+                ->option
+                ?->titles()
+                ->whereIn('store_id', [0, config('rapidez.store')])
+                ->first()
+                ->title
+        )->shouldCache();
+    }
+
+    // It would be nice if we could make a HasMany relation from this so it's possible
+    // to eager load it but DB::raw() to do the explode within SQL can't be used.
+    protected function option(): Attribute
+    {
+        return Attribute::make(
+            get: fn () => config('rapidez.models.product_option')::find(explode('_', $this->code)[1])
```

Comment:
In this case we do not need explode specifically, we could use something like
```sql
SUBSTRING_INDEX(SUBSTRING_INDEX(`code`, '_', -2), '_', 2)
```
This removes everything before the first occurence of `_` and then removes everything after the next occurence of `_`
emulating the behavior of `explode('_', $this->code)[1]`

Reply:
Sadly we can't use raw SQL within Eloquent relations + this will be removed when we migrate the cart to GraphQL